content/guides/webhook-subscriptions.md

```diff
@@ -0,0 +1,142 @@
+# Webhook subscriptions
+
+1. TOC
+{:toc}
+
+## Preface
+
+Periodically fetching data from API to keep all the objects in sync is not the most efficient solution. Ideally, data synchronization would be triggered by some specific event.
+
+Thanks to webhook subscriptions, you can be notified when some event occurs. You just need to select which events you want to subscribe to.
+
+## Requirements
+
+None, if you are an API partner and you've already registered your application, there is nothing else you need to do.
+
+## Webhooks: getting started
+
+### Application setting page
+
+First, go to [partner login page](https://www.bookingsync.com/en/partners/login).
+
+Once you are logged in, locate the application where you want to subscribe to webhooks and click the "Manage" button.
+
+### The "Webhooks" tab
+
+You should see a "Webhooks" tab
```

Comment:
@Azdaroth I don't see the "Webhooks" tab. Is it released? checked the branches on core none seem to be about webhooks.

Reply:
@kaizencodes It's currently hidden behind `webhooks_enabled` flag, we need the docs to get rid of it :P